Card.cs

```diff
@@ -0,0 +1,68 @@
+﻿using EnumColors;
+using System;
+
+namespace Task1_GUI
+{
+    internal class Card : IComparable
+    {
+        private int setterNominal;
+        private Colors.SevenColors setterColor;
+        public int Nominal { get; }
+        public Colors.SevenColors Color { get; }
+        public static int NumOfWinner { get; set; }
+        public Card(string combination)
+        {
+            CheckEnteredCombination(combination);
+            Nominal = setterNominal;
+            Color = setterColor;
+        }
+        public bool Equals(Card card)
+        {
+            return (Nominal == card.Nominal && Color == card.Color);
+        }
+        public override string ToString()
+        {
+            return (Nominal + " " + Color);
+        }
+        private void CheckEnteredCombination(string combination)
+        {
+            const int nominalPosition = 0, colorPosition = 2;
+            int enteredNominal = Convert.ToInt32(combination[nominalPosition]) - 48;
```

Comment:
I don't like some magic numbers in the code. Imagine that I don't know what does 48 mean. Move it to the const with some meaningful name.